lib/find_retryables.ts

```diff
@@ -33,7 +33,7 @@ const main = async () => {
 
   // Adding your Obit chain as a custom chain to the Arbitrum SDK
   try {
-    addCustomChain({ customChain: xai })
+    addCustomNetwork({ customL2Network: xai })
```

Comment:
Maybe more of an SDK review question, but shouldn't this be `customL3Network`?

Reply:
This is again a temporary misnomer, `L2Network` really means `ArbitrumNetwork, so you could either do L2 or L3